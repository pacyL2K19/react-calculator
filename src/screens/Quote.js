/* eslint-disable max-len */
import { Box } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
  textParagraph: {
    fontSize: 20,
    marginTop: theme.spacing(6),
  },
}));

const Quote = () => {
  const classes = useStyles();

  return (
    <Box width={1 / 2} m="auto">
      <p align="center" className={classes.textParagraph}>
        Mathematics is not about numbers, equations, computations,
        or algorithms: it is about understanding. â€“William Paul Thurston
      </p>
    </Box>
  );
};

export default Quote;
