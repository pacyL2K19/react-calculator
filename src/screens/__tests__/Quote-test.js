import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe('Quote page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Quote />);
    expect(tree).toMatchSnapshot();
  });
});
