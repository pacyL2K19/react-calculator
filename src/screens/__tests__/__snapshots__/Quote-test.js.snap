// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page should match the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <p
    align="center"
    className="makeStyles-textParagraph-1"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </p>
</div>
`;
